site/js/site.js

```diff
@@ -95,6 +95,7 @@ function onSiteDependenciesLoaded() {
    *  - `domain`: x axis possible values (see examples below)
    * - `yAxis`: same options as `xAxis`
    * - `disableZoom`: true to disable translation/scaling on the graph
+   * - `centerAxes`: true to center the axes in the middle of the graph
```

Comment:
Comment about API design, what about making this an option of the xAxis, yAxis fields like `      position: 'sticky'` e.g.

```
    xAxis: {
      label: 'x - axis',
      position: 'sticky',
    },
    yAxis: {
      label: 'y - axis'
      position: 'sticky',
    },
```

- Why `sticky`? Seems like a value that matches what the position field in CSS https://developer.mozilla.org/en-US/docs/Web/CSS/position
- What should be the default value for position? Given that this is a new option it should have a default that's backward compatible, possible values could be `['fixedBottom', 'fixedLeft', 'sticky']` with the defaults:

```
    xAxis: {
      label: 'x - axis',
      position: 'bottom',
    },
    yAxis: {
      label: 'y - axis'
      position: 'left',
    },
```